token-swap/program/src/instruction.rs

```diff
@@ -52,11 +52,11 @@ pub enum SwapInstruction {
     ///   0. `[]` Token-swap
     ///   1. `[]` $authority
     ///   2. `[writable]` token_(A|B) SOURCE Account, amount is transferable by $authority,
-    ///   4. `[writable]` token_(A|B) Base Account to swap INTO.  Must be the SOURCE token.
-    ///   5. `[writable]` token_(A|B) Base Account to swap FROM.  Must be the DEST token.
-    ///   6. `[writable]` token_(A|B) DEST Account assigned to USER as the owner.
-    ///   7. '[]` Token program id
-    ///   userdata: SOURCE amount to transfer, output to DEST is based on the exchange rate
+    ///   3. `[writable]` token_(A|B) Base Account to swap INTO.  Must be the SOURCE token.
+    ///   4. `[writable]` token_(A|B) Base Account to swap FROM.  Must be the OUTPUT token.
+    ///   5. `[writable]` token_(A|B) OUTPUT Account assigned to USER as the owner.
+    ///   6. '[]` Token program id
+    ///   userdata: SOURCE amount to transfer, output to OUTPUT is based on the exchange rate
```

Comment:
What motivated the change to `OUTPUT` here?
If we want to go this route, let's use output consistently. As it is, the js still uses `destination` and the rust `swap()` instruction method uses `dest` (which we should probably spell out, even though we do the same thing in spl-token)

Reply:
That's a good point.  I was mostly getting confused with the `into` and `from` accounts, and in the process I made those changes as well to clarify which was which.  I'll revert `output` to `destination` and expand where we only have `dest`.  From there, you'll be able to enjoy 0.75.0 :smile: